installer/manifests/keptn/charts/control-plane/templates/api-gateway-nginx.yaml

```
@@ -43,6 +43,11 @@ data:
         '' close;
       }
 
+      map $http_x_forwarded_proto $forwardproto{
```

Can you do this?
```suggestion
      map $http_x_forwarded_proto $forward_proto{
```

If not, you need to add `forwardproto` to `expect.txt`

---

Thx for the suggestion, I will update the PR .!